packages/patternfly-4/react-virtualized-extension/build/copyStyles.js

```diff
@@ -1,7 +1,8 @@
-/* eslint-disable no-case-declarations */
+/* eslint-disable no-case-declarations, @typescript-eslint/no-var-requires */
 const { copySync, readFileSync, writeFileSync } = require('fs-extra');
 const { resolve, dirname, join } = require('path');
 const { parse: parseCSS, stringify: stringifyCSS } = require('css');
+/* eslint-enable @typescript-eslint/no-var-requires */
```

Comment:
This is a build script. Not sure we need to apply linter here? Perhaps a different config for js?